pkg/types/validation/installconfig.go

```diff
@@ -699,3 +703,35 @@ func validateFIPSconfig(c *types.InstallConfig) field.ErrorList {
 	}
 	return allErrs
 }
+
+// validateCapabilities checks if additional, optional OpenShift components are specified in the
+// install-config to be included in the installation.
+func validateCapabilities(c *types.Capabilities, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	allCapabilitySets := sets.NewString()
+	allAvailableCapabilities := sets.NewString()
+	// Create a map and slice of *all* available sets and capabilities across all capability sets
```

Comment:
```suggestion
	// Create sets of all capability sets and *all* available capabilities across those capability sets
```

Reply:
I was just going through this file and had changed it before I saw this. I'll use your suggestion